app/views/paye/PayeGovernmentSpendingView.scala.html

```diff
@@ -51,16 +51,27 @@ <h2 class="heading-xlarge flush--top">@Html(Messages("paye.generic.tax_year_head
 		<dd class="value total"><strong>&pound;@viewModel.totalAmount</strong></dd>
 	</dl>
 
+@if(!isWelshTaxPayer){
```

Comment:
Can we inverse this?  !checks are harder to read

Reply:
Yes we can:)
done